README.md

```diff
@@ -72,22 +91,49 @@ Once you have the .AppImage file extracted, you can either **double click** the
+
+## License
 
-`User wants to open a channel`
+This project is open source under the MIT license, which means you have full access to the source code and can modify it to fit your own needs. See [LICENCE](LICENCE) for more information.
 
-`User wants to create a payment request`
+[MIT](LICENSE) © Jack Mallers
 
-`User wants to make a payment`
+[issues]: https://github.com/LN-Zap/zap-desktop/issues
+[releases]: https://github.com/LN-Zap/zap-desktop/releases
+[slack]: https://join.slack.com/t/zaphq/shared_invite/enQtMzMxMzIzNDU0NTY3LTgyM2QwYzAyZTA5OTAyMjEwMTQxZmZmZmZkNWUzMTU2MmMyNmMxNjY4Y2VjY2FiYTRkMTkwMTRlMTE4YjM2MWY
+[covenant]: http://contributor-covenant.org/version/1/3/0/
```

Comment:
You can drop this line. After that I think we're good...

Reply:
Have removed that unneeded reference. Now. I also updated ADVANCED.md and README.md  to use reference links for the slack and issues links so that they are consistently detailed in a single location at the bottom of the files.